File: src/main/java/com/acevedosharp/entities/EmpleadoDB.java
package com.acevedosharp.entities;

import javax.persistence.*;

@Entity
@Table(name = "empleado", schema = "app")
public class EmpleadoDB {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private @Id @Column(name = "empleado_id") Integer empleadoId;
    private @Column(name = "nombre") String nombre;
    private @Column(name = "telefono") String telefono;

    public EmpleadoDB() {
    }

    public EmpleadoDB(Integer empleadoId, String nombre, String telefono) {
        this.empleadoId = empleadoId;
        this.nombre = nombre;
        this.telefono = telefono;
    }

    public Integer getEmpleadoId() {
        return empleadoId;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }
}

File: src/main/kotlin/xyz/acevedosharp/persistence_layer/entities/EmpleadoDB.kt
package xyz.acevedosharp.persistence_layer.entities

import javax.persistence.*

@Entity
@Table(name = "empleado", schema = "epic")
class EmpleadoDB (
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "empleado_id")
    var empleadoId: Int? = null,

    @Column(name = "nombre")
    var nombre: String,

    @Column(name = "telefono")
    var telefono: String
)
